util/dvsim/SimCfg.py

```diff
@@ -41,6 +42,49 @@ def pick_wave_format(fmts):
     return fmt
 
 
+class Results:
+    '''An object wrapping up a table of results for some tests
+
+    self.table is a list of TestResult objects, each of which
+    corresponds to one or more runs of the test with a given name.
+
+    self.fail_msgs is a list of error messages, one per failing run.
+
+    '''
+    def __init__(self, items):
```

Comment:
Need to be able to handle the case where items is None (in case you end up mistakenly running a non-existent test).

Reply:
I don't think that `items` can ever be None. Looking at the code, it's always a list (`self.deploy` in `SimCfg._gen_results`, which is in turn set to a list in `SimCfg._create_deploy_objects`).